From: Andrea Bolognani <eof@kiyuko.org>
Date: Tue, 24 Mar 2009 15:57:12 +0100
Subject: Change default programs

x-terminal-emulator is the best default for a terminal
emulator on Debian, and xlock is not available at all.

Forwarded: not-needed
---
 spectrwm.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/spectrwm.c b/spectrwm.c
index 0737202..63c5ab3 100644
--- a/spectrwm.c
+++ b/spectrwm.c
@@ -11796,10 +11796,10 @@ validate_spawns(void)
 static void
 setup_spawn(void)
 {
-	setconfspawn(SWM_ASOP_BASIC, "lock", "xlock", 0, NULL);
-	setconfspawn(SWM_ASOP_BASIC, "term", "xterm", 0, NULL);
+	setconfspawn(SWM_ASOP_BASIC, "lock", "slock", 0, NULL);
+	setconfspawn(SWM_ASOP_BASIC, "term", "x-terminal-emulator", 0, NULL);
 	setconfspawnflags(SWM_ASOP_BASIC, "term", "xterm_fontadj", 0, NULL);
-	setconfspawn(SWM_ASOP_BASIC, "spawn_term", "xterm", 0, NULL);
+	setconfspawn(SWM_ASOP_BASIC, "spawn_term", "x-terminal-emulator", 0, NULL);
 	setconfspawnflags(SWM_ASOP_BASIC, "spawn_term", "xterm_fontadj", 0,
 	    NULL);
 
